types.ts

```diff
@@ -103,3 +103,9 @@ export interface GitLabRepository {
     }[];
   };
 }
+
+export interface Data {
+  repositories: Repository[];
+  languages: CountableTag[];
```

Comment:
Maybe we could update this to `CountableLanguage` for consistency, but not a hard requirement right now.

Reply:
I've implemented this.